plugins/modules/system/lvol.py

```diff
@@ -136,6 +137,12 @@
     lv: test
     size: +100%FREE
 
+- name: Extend the logical volume by given space
+  lvol:
```

Comment:
```suggestion
  community.general.lvol:
```
(See the other examples.)